solution/ui/regulations/eregs-component-lib/src/eventbus/index.js

```diff
@@ -0,0 +1,3 @@
+import mitt from "mitt";
+
+export default mitt();
```

Comment:
followed `mitt` eventbus recipe here: 

https://binodmahto.medium.com/event-bus-implementation-in-vue-5eb194aed569